resources/basics.py

```diff
@@ -269,6 +269,14 @@ def metadata_exomes_tsv_path(version=CURRENT_EXOME_META):
     return 'gs://gnomad/metadata/exomes/gnomad.exomes.metadata.{0}.tsv.bgz'.format(version)
 
 
+def metadata_genomes_ht_path(version=CURRENT_GENOME_META):
```

Comment:
Given that we now have streamlined metadata available, e.g., `gs://gnomad/sample_qc/input_meta/gnomad.genomes.streamlined_metadata.2018-03-21.txt.bgz`, is there still a need to have the old metadata floating around? Can we remove these?

Reply:
That's fine, I guess I just meant the data files that are currently there (2017-06-02). Some of them are already hidden in the `sample_qc/input_meta/source` directory so it's weird that they are also "out there" to be seen.

Reply:
Yeah, I mean, that's fine. I'd probably keep the ones that are in there - no harm in having a few, and then the CURRENT_META_VERSION points to the newer ones.